src/riscv.h

```diff
@@ -571,6 +573,10 @@ typedef struct {
 
     /* plic object */
     plic_t *plic;
+
+    /* virtio-blk device */
+    uint32_t *disk;
+    virtio_blk_state_t *vblk;
```

Comment:
Ok it seems like UART and VIRTIO_BLK come from semu, and PLIC may have a little bit different design here......

In my opinion, I am just thinking of wrapping these things in `virtio_t`, but maybe we can refactor these in other PR. This is not a critical issue.